src/components/SearchPage.js

```diff
@@ -181,7 +181,7 @@ class SearchPage extends React.Component {
                                    getsSelectedLanguage = { this.handleSelectCol }
                                    allLanguages = { this.state.selectedColumns }/>
 
-                        <RowsPerPageToggleButton darkMode= {this.props.darkMode} onButtonClicked = { this.handleRowsPerPageChanges }/>
+                        <RowsPerPageToggleButton darkMode= {this.props.darkMode} onButtonClicked = { this.handleRowsPerPageChanges } activebtn = { this.state.rowsPerPage }/>
```

Comment:
one small thing, can we use camelcase for prop name?

Reply:
sry mb